OrbitQt/orbitmainwindow.cpp

```diff
@@ -318,6 +337,20 @@ void OrbitMainWindow::OnNewTopDownView(std::unique_ptr<TopDownView> top_down_vie
 
 void OrbitMainWindow::OnNewSelectionTopDownView(
     std::unique_ptr<TopDownView> selection_top_down_view) {
+  if (selection_top_down_view->child_count() > 0) {
+    ui->RightTabWidget->setTabEnabled(ui->RightTabWidget->indexOf(ui->selectionTopDownTab), true);
+    if (ui->RightTabWidget->currentWidget() == ui->topDownTab) {
```

Comment:
If I see this correctly, then this logic here has to be exactly complementary to the logic in OnNewSelectionReport, or otherwise which tab is selected will depend on the order in which OnNewSelectionReport and OnNewSelectionTopDownView are called (they are both called on a new selection, right?). 

If my understanding is correct, I would add a comment to explain this and point out the connection. O/w someone might "optimize" the code here or above later without fully understanding the implications.

Reply:
If I got your question correctly, yes, this condition and the one in `OnNewSelectionReport` need to work together, I added a (hopefully not too convoluted) comment to both functions.

Reply:
Sounds good, yes, that captures what I had in mind.